.github/workflows/release.yml

```diff
@@ -176,7 +186,7 @@ jobs:
           path: frontend/dist
 
       - name: Build Docker Image
-        run: docker build . --file Dockerfile --tag ${{ env.ACR_REPO }}/${{ env.PREFIX }}:latest
+        run: docker build . --file Dockerfile --tag ${{ env.ACR_REPO }}/${{ env.PREFIX }}:latest --tag ${{ env.ACR_REPO }}/${{ env.PREFIX }}:${{ github.run_id }}
```

Comment:
It's annoying we have to repeat the steps for each environment, but to access GitHub deployment environment variable, the environment cannot be parameterized to prevent secrets hijacking.